### PR TITLE
Tours buttons behavior

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose

Considering the buttons behavior of the 3 Tours from the improvement https://jira.autodesk.com/browse/DYN-5404

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang 

### FYIs
@RobertGlobant20

### CODE REVIEW

**Comment by jesusalvino** [comment]
Tours_Enabling

**Comment by jesusalvino** [comment]
The test WorkspaceContextMenu_IfSubmenuOpenOnMouseHover passes locally

imagen

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs [hunk 1]
```diff
@@ -2529,6 +2529,7 @@ private void PackagesMenuGuide_Click(object sender, RoutedEventArgs e)
             try
             {
                 dynamoViewModel.MainGuideManager.LaunchTour(GuidesManager.PackagesGuideName);
+                dynamoViewModel.OnEnableShortcutBarItems(false);
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs:2532]: Just like how you implemented above, can you put this call as part of `LaunchTour` call?

Comment (jesusalvino) [reply src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: Done.

**Review by QilongTang — APPROVED** [review]

**Pull request merged**